cubeviz/image_viewer.py

```diff
@@ -242,6 +242,19 @@ def __init__(self,  *args, cubeviz_layout=None, **kwargs):
         self._hub.subscribe(self, WavelengthUnitUpdateMessage, handler=self._update_wavelength_units)
         self._hub.subscribe(self, FluxUnitsUpdateMessage, handler=self._update_flux_units)
 
+    @property
+    def cubeviz_unit(self):
+        return self._cubeviz_unit
+
+    @cubeviz_unit.setter
+    def cubeviz_unit(self, cubeviz_unit):
```

Comment:
@robelgeda Might want to change the input parameter or function name. Never know in the future if there might be some weird collisions if the parameter and function name are the same.  (Or change the function name?)